mlops/london_taxi/src/mlops_pipeline.py

```diff
@@ -284,9 +284,9 @@ def prepare_and_execute(
         config.aml_config["subscription_id"],
         config.aml_config["resource_group_name"],
         config.aml_config["workspace_name"],
-        config.environment_configuration["env_base_image"],
-        pipeline_config["conda_path"],
-        pipeline_config["aml_env_name"],
+        env_base_image_name=config.environment_configuration["env_base_image"],
```

Comment:
i have no objections, but the dstoolkit-v2 team may ask why the change in syntax.

Reply:
I named it back.. no need to change it. good catch